themes/V3/5ePHB/style.less

```diff
@@ -976,24 +854,3 @@ h6,
 		}
 	}
 }
-// *****************************
-// *       INDEX
-// *****************************/
-.page {
-	.index {
-		font-size : 0.218cm;
```

Comment:
Another one where BLANK probably shouldn't set a specific font size.

Reply:
Removed in Blank